src/api/app/services/rpmlint_log_parser.rb

```diff
@@ -0,0 +1,53 @@
+class RpmlintLogParser
+  attr_reader :errors, :badness, :warnings, :info
+
+  def initialize(content:)
+    @content = content || ''
```

Comment:
You can just initialize the default value in params. 

```
def initialize(content: '')
   @content = content
   ...

```

Reply:
True! Applied.